navigation-branch/navigation.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation</title>
+    <link rel="stylesheet" href="./navigation.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <h1> Navigation </h1>
+    <h2> Desktop (Simple) </h2>
+    <section class="container-for-nav">
+        <div class="container-logo">
+            <a href="#0"> <img src="https://picsum.photos/id/1024/50/50" alt="company logo"></a>
+        </div>
+        <nav class="actual-navigation">
+            <ul class="container-navigation-items">
+                <a href="#0">
+                    <li> Home </li>
+                </a>
+                <a href="#0">
+                    <li> About </li>
+                </a>
+                <a href="#0">
+                    <li> Services </li>
+                </a>
+                <a href="#0">
+                    <li> Products </li>
+                </a>
+                <a href="#0">
+                    <li> Contact </li>
+                </a>
+            </ul>
+        </nav>
+    </section>
+
+    <!-- Ratings below -->
+    <section class="container-for-rating">
+        <h2> Rating</h2>
+        <div class="general-rating rating-with-number">
+            <span class="rating-rank"> 4.1
+                <i class='fa fa-star fa-md rating-checked'></i>
+            </span>
+        </div>
+
+        <h2>Star Rating</h2>
+        <div class="general-rating rating-only-stars">
+            <i class='fa fa-star fa-lg'></i>
+            <i class='fa fa-star fa-lg'></i>
+            <i class='fa fa-star fa-lg'></i>
+            <i class='fa fa-star fa-lg'></i>
+            <i class='fa fa-star fa-lg'></i>
+        </div>
+    </section>
+
+</body>
+
+</html>
```

Comment:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file Preferrable to have empty line at eof

Reply:
![forArunesh](https://user-images.githubusercontent.com/67211994/152049132-55791089-b7c2-45a5-b3ad-1b5be76c18a9.JPG)

1. It was a very good suggestion and I thank you for providing it.
2. I found this in the same article (attached as image)
2. On using **'beautify' extension,** last empty line is deleted on it own
3. For now, I will leave it as it is. But open to change it as I learn more on this topic :)

Reply:
Also the article dates back to some 11 years 11 months ago as of today. This was asked almost 12 years ago!